mzbench/scenarios/fan_in.bdl

```diff
@@ -2,13 +2,13 @@
 
 #######
 # Scenario:
-# A single subscriber reading from "<MF_CHANNEL>/#" topic
-# 10000 publisher publishing to exclusive topic "<MF_CHANNEL>/<client_id>"
+# A single subscriber reading from "channels/<MF_CHANNEL_ID>/messages/#" topic
+# 10000 publisher publishing to exclusive topic "channels/<MF_CHANNEL_ID>/messages/<client_id>"
 # Overall msg rate: 10 000 msg/s
 # Message size: 100 random bytes
 # Running time: 5 min
 # QoS level: 2
-# NOTE: Update values in test, according to your values
+# NOTE: Update values in defaults or set environment variables according to your values
 # <MF_MQTT_ENPOINT> = IP/domain of your MQTT endpoint
 # <MF_THING_1_ID> = First thing ID of pre-provisioned Mainflux thing
```

Comment:
I do not see `MF_THING_1_ID` used.

Reply:
I've mistakenly named them `MQTT_THING_` instead `MF_THING_` in the scenario.
Thank, renamed.